e2e/tests/functional/search.e2e.spec.js

```diff
@@ -139,6 +139,45 @@ test.describe("Search Tests @unstable", () => {
         await expect(searchResults).toHaveText(folderName);
     });
 
+    test('Search results are debounced @couchdb', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6179'
+        });
+        //Go to baseURL
+        await page.goto("./", { waitUntil: "networkidle" });
+
+        await createObjectsForSearch(page);
+
+        let networkRequests = [];
+        page.on('request', (request) => {
+            const searchRequest = request.url().endsWith('_find');
+            const fetchRequest = request.resourceType() === 'fetch';
+            if (searchRequest && fetchRequest) {
+                networkRequests.push(request);
+            }
+        });
+
+        // Full search for object
+        await page.type("input[type=search]", 'Clock', { delay: 100 });
+
+        // wait for debounced search requests to accumulate and send
+        await new Promise(resolve => setTimeout(resolve, 500));
+
+        // Wait for search to complete
+        await page.waitForLoadState('networkidle');
```

Comment:
We should try to avoid waits on networkidle as much as possible. Could we wait for the search results to be visible instead?

edit: I did some research and right now, it looks like the structure of the `SearchResultsDropdown` html is not super conducive to this. Let's chat about this when you have some time, I think we can make this way more deterministic with slight modifications to the html. We can also fix the other tests in this suite and stabilize them so they can run in CI